### PR TITLE
monitor block cache usage

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
Monitor rocksdb block cache usage.
@siddontang @BusyJay @hhkbp2 PTAL

### CODE REVIEW

File: src/raftstore/store/engine_metrics.rs [hunk 1]
```diff
@@ -203,6 +208,13 @@ lazy_static!{
             &["type"]
         ).unwrap();
 
+    pub static ref STORE_ENGINE_BLOCK_CACHE_USAGE_GAUGE_VEC: GaugeVec =
+        register_gauge_vec!(
+            "tikv_engine_block_cache_size_bytes",
+            "Usage of each column families' block cache.",
+            &["type"]
```
Comment (siddontang) [inline src/raftstore/store/engine_metrics.rs]: is "cf" better than "type" here?

**Comment by hhkbp2** [comment]
LGTM

File: src/raftstore/store/engine_metrics.rs (hunk 1 above)
Comment (siddontang) [inline src/raftstore/store/engine_metrics.rs]: any update?

**Comment by siddontang** [comment]
LGTM

**Pull request merged**